docker-centos/Dockerfile

```diff
@@ -16,6 +16,10 @@ rpm -V docker-rhel-push-plugin && \
 mkdir -p /exports/hostfs/etc/docker && \
 yum clean all
 
+# Move the /etc/sysconfig/docker out of the way as daemon.json is the proper config location
```

Comment:
upgrade should be already managed no?  atomic should not overwrite `daemon.json` when it is already present, and with this change the `/etc/sysconfig/docker` is renamed.  Altough, I think it is enough to drop it from the systemd unit file for it to be ignored from the docker system container at startup